web-app-sql-docdb-search/README.md

```diff
@@ -0,0 +1 @@
+# Provision a web app with a SQL Database, DocumentDB and Azure Search

```

Comment:
Yep, you can generate the buttons before merging. Also make sure it's "Azure" instead of "azure" since Github URLs are case sensitive